### ISSUE TITLE
Freiburg forest dataset :zero loss , no variables to save & low mIoU .

### ISSUE THREAD

**Issue by onizuka94** [DeepSceneSeg/AdapNet-pp#16]
My environment : python 3.6.7 and Tesnorflow 1.10 . 

- I augmented and resized the annotated images within the freiburg dataset and was able to generate train 70000 RGB images and their segmented ground truth labels .

- generated two text files (train and test ) containing full paths as mentioned in the README

- created the tf_records_train file and the tf_records_test file .

- Resolved the python2 -> python3 errors .

I run train.py with the initialization checkpoint provided : I always get loss = 0 . 

I change the config file and feed in the pre-trained checkpoint (the one where RGB mIoU = 83.18) and I replace in line 70 and 71 var.name with var.name.replace(':0','') in train.py then run it : I get No variables to save (yes I'm sure of the pretrained chkpt path , I've even put it in the init folder next to the init chkpt ) 

I try to run evaluate.py with : init chkpt , the pretrained model.chkpt you provided , and a checkpoint of a training that has been done on 3400 imges (still going to test if I train the 70K images that will change something ) and I get low mIoU of respectively 8.xxx , 8.xxx and 10.xxxx and this has been tested on both 20K test dataset and a few hundreds of images dataset , yet resulting in the same results . 

I've been stuck with this for days , and I would like any clear explanation on what I missed or any fix the issue .

**Comment by avalada** [comment]
init chkpt is just the weights of resnet_v2 trained on imagenet (so miou will be low).
I'll suggest before you train your own model, make sure that the pre-trained model's checkpoint prediction is same as reported.
Can you make sure your gt labels are mapped correctly? It looks like there might be something wrong with the training data.
